hexrd/ui/hand_drawn_mask_dialog.py

```diff
@@ -89,6 +99,7 @@ def axes_entered(self, event):
             return
 
         self.ax = event.inaxes
+        self.det = self.ax.get_title()
```

Comment:
The hand drawn mask works also in the polar view, right? What does this do in the polar view?

Reply:
This is just moving the existing logic that was [here](https://github.com/HEXRD/hexrdgui/blob/master/hexrd/ui/hand_drawn_mask_dialog.py#L134) so that the information is grabbed when the masked is created rather than when it is completed (as this may have happened after the mode has changed). For the polar view no title is returned but since it is not used unless we're in the raw view (see [signal](https://github.com/HEXRD/hexrdgui/blob/master/hexrd/ui/main_window.py#L724-L725) and [function](https://github.com/HEXRD/hexrdgui/blob/master/hexrd/ui/main_window.py#L727-L745)) this does not matter.

Reply:
I see, that's good to know! Can you leave a short comment here about that?

Reply:
Added :+1: